src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/controller/RecallControllerTest.kt

```diff
@@ -103,13 +105,76 @@ class RecallControllerTest {
     assertThat(results.body, equalTo(recallResponse.copy(recallId = recall.recallId(), createdByUserId = userUuid)))
   }
 
+  private val bookedByUserId = ::UserId.random()
+  private val assignee = ::UserId.random()
+  private val dossierCreatedByUserId = ::UserId.random()
+
+  private val beingBookedOnRecall = Recall(::RecallId.random(), NomsNumber("lf34546"), createdByUserId, now, FirstName("Barrie"), null, LastName("Badger"))
```

Comment:
Pretty hard to see wood for trees here.
Created these Recall instances in a test helper method?
Also do the NOMS numbers need any relation to one another?  If not then they can all be randomNoms?